src/objectTranslations/Contact-es.objectTranslation

```diff
@@ -3473,19 +3473,35 @@
         <name>PreferredPhone__c</name>
         <picklistValues>
             <masterLabel>Home</masterLabel>
-            <translation>Particular</translation>
+            <translation>Teléfono particular</translation>
         </picklistValues>
         <picklistValues>
             <masterLabel>Mobile</masterLabel>
-            <translation>Móvil</translation>
+            <translation>Teléfono Móvil</translation>
         </picklistValues>
         <picklistValues>
             <masterLabel>Other</masterLabel>
-            <translation>Otros</translation>
+            <translation>Otro teléfono</translation>
```

Comment:
I'm worried in all of these translations that maybe we're guessing and we should actually leave this up to the translations team or at least run it by them first. 

For example, why is Other / Other Phone "Otro telefono", whereas all of the others have "Telefono ..." first?

(I don't know any other languages so it's possible that this is correct, but I'm questioning it...)

Thoughts, @skosgi1 @ceiroa?

Reply:
Yeah looks like one is "other" and the other one translates to "another" ? but they have to match the field labels anyway for the functionality to work, so not sure about the inconsistency here...

Reply:
Well, we don't have to guess if we are taking the value from the label. 

Also, yeah, it is correct that in this case "teléfono" goes after and the others have it first.

Reply:
@skosgi1 haha! I'm sorry. I forgot that we were mapping these to the actual field labels, which *are* translated already by Salesforce. Nevermind!